modules/reactor/src/meshgenerators/PinMeshGenerator.C

```diff
@@ -189,9 +189,9 @@ PinMeshGenerator::PinMeshGenerator(const InputParameters & parameters)
     _has_block_names = false;
 
   // Initial block id used to define radial regions of pin
-  unsigned int pin_block_id_start = 10000;
+  subdomain_id_type pin_block_id_start = 10000;
   // Use special block id to designate TRI elements
-  unsigned int pin_block_id_tri = pin_block_id_start - 1;
+  subdomain_id_type pin_block_id_tri = pin_block_id_start - 1;
```

Comment:
@shikhar413 Any insight on this? Thanks.

Reply:
@shikhar413 Any insight for this, thanks.

Reply:
I'm not sure I understand the question, but 10000 was chosen as a default starting value for setting the initial block IDs in pin mesh generator, and then incrementing by one for each radial region (ring / background / duct) in the pin region. Each radial region needs to have a unique radial block ID so that the element can be mapped to the correct region ID extra element ID in generate(). Eventually in generate(), these block IDs are also collapsed and remapped so that there is ultimately one block ID for the quad elements of the pin and one block ID for the tri elements of the pin. The logic of setting the axial and radial region ID map while keeping two block IDs to describe the entire mesh can't be done just by setting the MG parameters, and a lot of this logic is coded into generate instead

Reply:
@GiudGiud Let me know if you have any additional questions about this, or I will resolve this one. Thanks.